osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -425,7 +426,7 @@ private partial class DateLabel : DrawableDate
             public DateLabel(DateTimeOffset date)
                 : base(date)
             {
-                Font = OsuFont.GetFont(size: 13, weight: FontWeight.Bold, italics: true);
+                Font = OsuFont.GetFont(size: 14, weight: FontWeight.Bold);
```

Comment:
I don't wanna get into it too much, likely for the same reasons as you, but I've got to admit that I'm skeptical that this change will really address anyone's concerns here. This just bumps the size of the date text by a point. The accuracy text size is not changed at all and like half the comments listed in https://github.com/ppy/osu/issues/32055#issuecomment-2677579485 listed accuracy as being problematic as well. And looking at the display again both these texts do seem a bit comically small in comparison to everything else.

Is there a specific reason that this isn't just a straight revert to the sizings before the team PR, i.e. size 17? Even 16 would be somewhat better probably.

Reply:
sure whatever let's just do that. it's silly though.